libsoftwarecontainer/component-test/softwarecontainerlib_componenttest.cpp

```diff
@@ -487,7 +501,7 @@ TEST_F(SoftwareContainerApp, TestUnixSocket) {
 
                 close(fd2);
 
-                return done == 1 ? EXISTENT : NON_EXISTENT;
+                return done == 1 ? SUCCESS : FAILURE;
```

Comment:
Would not just returning done be enough in this case? If not then the job should be rewritten in such a way that it does.